deepspeed/runtime/zero/constants.py

```diff
@@ -29,7 +29,8 @@
     "cpu_offload_use_pin_memory": [true|false] (deprecated),
     "sub_group_size" : 1000000000000,
     "offload_param": {...},
-    "offload_optimizer": {...}
+    "offload_optimizer": {...},
+    "stage2_find_unused_parameters": [true|false]

```

Comment:
Can you please just name this "find_unused_parameters" so it can be easily extended to other stages in future.

Reply:
No problem, now it changed.